### ISSUE TITLE
Chronic inflammation

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6189]
**Preferred term label:**
Chronic inflammation

**Synonyms**
Chronic increased inflammatory response

**Definition (free text, please give PubMed ID)**
A slow, long-term inflammation lasting for prolonged periods of several months to years.
PMID: 29630225 

**Parent term (use hpo.jax.org/app)**
Increased inflammatory response

**Comment by pnrobinson** [comment]
I think that this is more of an inference rather than a direct observation. Perhaps it would be better to use the "Chronic" modifier with one of the subterms of ``Increased inflammatory response`` -- otherwise, we would need to add a "Chronic' version of each of the subterms.
If there is a use case that cannot be implemented in this way, please reopen this issue or contact me.
